src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -60,6 +62,14 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
 
   constructor(private readonly context: vscode.ExtensionContext) {}
 
+  private updateWebview(document: vscode.TextDocument, webviewPanel: vscode.WebviewPanel) {
+    this._oneConfig = ini.parse(document.getText());
```

Comment:
#### For reviewers

`this` keyword is not recognized with previous code.
That's why this function is extracted as member function.